src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,36 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    loading: false,
+    errorMessage: '',
+    categories: [],
+  },
+  mutations: {
+    doneGetAllCategories(state, payload) {
+      state.categories = [...payload.categories].reverse();
```

Comment:
[nits]

以下のように修正をお願いします！

```suggestion
      state.categories = [...payload.categories.reverse()];
```

特に大きな差はないと思うのですが、シンプルにどこまでを`[]`で囲んで新しい配列にするかの基準が特にないので、元の配列を操作したものから新しい配列を作成するとしておけば良いかなと思います！

Reply:
043f65837c56c2a63e79bb904b5056a25f4806b1

確かに `[ ]` のなかに入れた方が良いですよね。
書き換えました！